### ISSUE TITLE
Bug: incorrectly complains about whitespace

### ISSUE THREAD

**Issue by spassvogel** [schoero/eslint-plugin-better-tailwindcss#144]
I'm using the `next/font/google` library to import google fonts into  my nextjs library.

typescript
```
import { DM_Sans, Figtree } from 'next/font/google'

const dmSans = DM_Sans({
  subsets: ['latin'],
  display: 'swap',
  variable: '--font-dm-sans',
})

const figtree = Figtree({
  subsets: ['latin'],
  display: 'swap',
  variable: '--font-figtree'
})
```

Then you add these fonts as classNames to your html element:

```
    <html className={`${dmSans.variable} ${figtree.variable}`} >

```

This is a quite common use case. If you spawn a nextjs app using `create-next-app` it will include Gheist and Gheist_Mono. However, the linter complains the space in the template string is not necessary.

<img width="805" alt="Image" src="https://github.com/user-attachments/assets/e5964c1f-8d51-472b-af8c-b52efda1c352" />

But it **is**, if you remove this space the font won't load!

**Comment by spassvogel** [comment]
There are some workarounds possible yeah but I think this rule is mafunctioning a bit...
